politeiawww/comments/process.go

```diff
@@ -321,6 +322,10 @@ func (c *Comments) processVotes(ctx context.Context, v v1.Votes) (*v1.VotesReply
 	}
 	commentVotePopulateUserData(cv, *u)
 
+	// Sort comment votes by timestamp.
```

Comment:
Not descriptive enough.  Are you sorting them from oldest to newest or vise versa?